README.md

```diff
@@ -42,14 +42,12 @@ This action deletes versions of a package from [GitHub Packages](https://github.
 
   # The number of old versions to delete starting from the oldest version.
   # Defaults to 1.
-  # Cannot be more than 100.
   num-old-versions-to-delete:
 
   # The number of latest versions to not delete.
-  # Defaults to 0.
-  # When this is set greater than 0 it will delete all deletable package versions except the specified no.
-  # `num-old-versions-to-delete` will not be taken into account with this option.
-  # Cannot be more than 100.
+  # This cannot be specified with `num-old-versions-to-delete`. By default, `num-old-versions-to-delete` is taken into account.
```

Comment:
Call it as This cannot be specified with `num-old-versions-to-delete`. By default,`num-old-versions-to-delete` takes precedence over `min-versions-to-keep`